xlsx/src/export/test/test_export.rs

```diff
@@ -78,6 +78,38 @@ fn test_values() {
     }
 }
 
+// This test verifies whether exporting the
+// merged cells functionality is happening
+// properly or not.
+
+// This test is bit unconventional from the 
+// other tests as it first loads the excell
+// having the merged cell and exports it to 
+// anothe xlsx and verifies whether merged
+// cell node is same in both of the xlsx file
+#[test]
+fn test_exporting_merged_cells(){
+    let expected_merge_cell_ref;
+    let got_merge_cell_ref;
```

Comment:
You don't need to declare  `got_merge_cell_ref` here. You can just do this at the point where it is defined. You do need to declare `expected_merge_cell_ref` because it is defined in a place with a smaller scope. One handsome thing you can do is:

```rust
    let expected_merged_cell_ref = {
        // loading the xlsx file containing merged cells
        let example_file_name = "tests/example.xlsx";
        let mut model = load_from_xlsx(example_file_name, "en", "UTC").unwrap();
        let merged_cell_ref = model.workbook.worksheets.get(0).unwrap().merge_cells.clone();
        // exporting and saving it in another xlsx
        model.evaluate();
        save_to_xlsx(&model, temp_file_name).unwrap();
        merged_cell_ref
    }
```

But that's not needed (I'm not even sure I like it!)

Reply:
Understood. This does look clean.